core/src/main/java/io/neow3j/transaction/TransactionBuilder.java

```diff
@@ -513,8 +524,27 @@ public NeoInvokeScript callInvokeScript() throws IOException {
      *                                           sender cannot cover the transaction fees.
      */
     public Transaction sign() throws Throwable {
+        return sign(neow3j.getNetworkMagic());
```

Comment:
are you using the `this.neow3j` pattern in other classes? if not, it's fine to leave like this. Otherwise, change it here.

Reply:
No, usually not.